### DISCUSSION TITLE
Intergration into Synology DS420+

### DISCUSSION THREAD

**Discussion by jimmytwopapers** [mathoudebine/turing-smart-screen-python#307] [General]
Have been mucking rounfd to see if i can output some stats from my NAS, 
Uing the docker and this git respostiroy, IT cant seem to find the COM PORT> 

Tried numrtous different config.yamls., To no avilble. 

Anyone know how to get more descriptvie info on comms in linix systems?>

**Comment by ghost** [comment]
The project requires that the kernel was built with the `cdc-acm` kernel module because that is the module responsible for generating the `ttyACM` interface that this program uses to communicate with the smart screen. If you are using an embedded Linux or similar without this support then you are out of luck. The kernel support is just not there.
